src/viam/sdk/robot/client.hpp

```diff
@@ -155,9 +197,10 @@ class RobotClient {
     std::shared_ptr<ViamChannel> viam_channel_;
     std::shared_ptr<Channel> channel_;
     bool should_close_channel_;
-    std::unique_ptr<RobotService::Stub> stub_;
+    struct Impl;
```

Comment:
@acmorrow a question for you! you proposed naming this struct `Impl` but that naively seems to go against our standard naming convention for structs (that they be `snake_case`). I realize this is a bit of a unique case in the SDK (`Impl` is private while `struct`s in general are defined by their public-ness, and this is our only current instance of the `pimpl` idiom). Is it more idiomatic given these distinctions to break the normal naming convention and call it `Impl`, or keep with the struct naming convention and call it `impl`?

Reply:
Oh, I'm just typing from muscle memory there. Fine to call it `impl` I think.

I truly wish we could bulk convert the whole SDK to `snake_case`.